rewrite-java-core/src/test/java/com/azure/recipes/core/v2/TextTranslationClientBuilderHttpTraitTest.java

```diff
@@ -0,0 +1,181 @@
+package com.azure.recipes.core.v2;
+
+import org.intellij.lang.annotations.Language;
+import org.junit.jupiter.api.Test;
+import org.openrewrite.test.RecipeSpec;
+import org.openrewrite.test.RewriteTest;
+
+
+import static org.openrewrite.java.Assertions.java;
+
+/**
+ * HttpTraitTest tests interface migration from com.azure.core.client.traits.HttpTrait
+ * to io.clientcore.core.models.traits.HttpTrait.
+ * Tests simple method renaming with declarative recipe.
+ * @author Annabelle Mittendorf Smith
+ */
+
+public class TextTranslationClientBuilderHttpTraitTest implements RewriteTest {
+    /**
+     * This method defines recipes used for testing.
+     * @param spec stores settings for testing environment.
+     */
+    @Override
+    public void defaults(RecipeSpec spec) {
+        spec.recipeFromResource("/META-INF/rewrite/rewrite.yml",
+                "com.azure.rewrite.java.core.MigrateAzureCoreSamplesToAzureCoreV2");
+    }
+
+    /**
+     * Test simple declarative rename of:
+     * retryOptions to httpRetryOptions
+     * pipeline to httpPipeline
+     * addPolicy to addHttpPipelinePolicy
+     * and complex rename of clientOptions to httpRedirectOptions
+     */
+    @Test
+    void test_methodsRenamed_successful() {
+        @Language("java") String before = "import com.azure.ai.translation.text.TextTranslationClient;\n" +
+                "import com.azure.ai.translation.text.TextTranslationClientBuilder;\n" +
+                "\n" +
+                "public class UserClass {\n" +
+                "\n" +
+                "    TextTranslationClient textTranslationClient = new TextTranslationClientBuilder()\n" +
+                "            .pipeline(null)\n" +
+                "            .addPolicy(null)\n" +
+                "            .retryOptions(null)\n" +
+                "            .buildClient();\n" +
+                "\n" +
+                "}\n";
+
+
+        @Language("java") String after = "import com.azure.ai.translation.text.TextTranslationClient;\n" +
+                "import com.azure.ai.translation.text.TextTranslationClientBuilder;\n" +
+                "\n" +
+                "public class UserClass {\n" +
+                "\n" +
+                "    TextTranslationClient textTranslationClient = new TextTranslationClientBuilder()\n" +
+                "            .httpPipeline(null)\n" +
+                "            .addHttpPipelinePolicy(null)\n" +
+                "            .httpRetryOptions(null)\n" +
+                "            .buildClient();\n" +
+                "\n" +
+                "}\n";
+
+        rewriteRun(
+                java(before,after)
+        );
+    }
+
+
+    @Test
+    void test_uninitializedParams_and_imports_changed() {
```

Comment:
We should probably use camel casing even for test methods here for Java convention.

Reply:
Fixed in commit [9364625](https://github.com/Azure/azsdk-java-rewrite-recipes/pull/16/commits/936462508c26abd77a1ec4cea422690d433768a2)